### PR TITLE
refactor(android): Change com.tavultesoft.kmea package to com.keyman.engine 🛋

### PR DESCRIPTION
**Pull request by darcywong00**
> If possible, @mcdurdin would also like to move the KMManager package 
> from `com.tavultesoft.kmea` to `com.keyman.engine.manager`

_Originally posted by @darcywong00 in https://github.com/keymanapp/keyman/issues/7881#issuecomment-1371761542_
      
There were too many uses of "protected" members in the com.tavultesoft.kmea classes, so I wasn't able to isolate KMManager.java into `com.keyman.engine.manager`.
So the next best thing I could do was mass-rename the package from `com.tavultesoft.kmea` to `com.keyman.engine`.
This rippled to the package names in KMEA Android Manifest.xml

But I'm intentionally leaving apps like "Keyman for Android" at `com.tavultesoft.kmapro` since they're published.

The only change that wasn't a rename was for some reason FileUtilsTest changed the contents in the logs from 3 items to 2.

## Breaking change for KMEA consumers

Consumers of Keyman Engine for Android will need to update their references from `com.tavultesoft.kmea` to `com.keyman.engine` accordingly.

## User Testing
This PR affected practically every Android source file, so check that all the Android apps load and behave correctly
Setup - For each test, load the PR build of the app on an Android device/emulator. 

* **TEST_KEYMAN** - Verifies Keyman for Android
1. Install and launch the PR build of "Keyman for Android"
2. Verify app loads fine and functions normally

* **TEST_KMSAMPLE1** - Verifies KMSample1 app
1. Install and launch the Android PR build of KMSample1 
2. Verify app loads fine and functions normally

* **TEST_KMSAMPLE2** - Verifies KMSample2 app
1. Install and launch the Android PR build of KMSample2 
2. Verify app loads fine and functions normally

* **TEST_FV_ANDROID** - Verifies FirstVoices for Android app
1. Install and launch the Android PR build of "FirstVoices for Android"
2. Verify app loads fine 
3. From the setup menu, install and enable the keyboard for BC Coast --> Sencoten
4. Allow time for the dictionary to also install in the background
5. Complete the setup menu so FV for Android is enabled as a system keyboard
6. Launch a separate app (e.g. Chrome) and select a text field to type in
7. With FV Android as the selected keyboard, verify it functions normally

### CODE REVIEW

**Comment by darcywong00** [comment]
Hmm, no CI artifacts for user-testing. I'll try closing an re-opening the PR

**Comment by bharanidharanj** [comment]
# Test Results

* **TEST_KEYMAN (OPEN):** Tested with the attached Keyman alpha-test-7981 build and in API 30 / Android 11 emulator and it working normally. 

* **TEST_KMSAMPLE1 (OPEN):** Installed KMSample1 build. Able to use the Tamil keyboard without having any issue. 

   

* **TEST_KMSAMPLE2 (OPEN):** Installed KMSample 2 build. Able to set the KMSample2 keyboard as the System Keyboard. 

   

* **TEST_FV_ANDROID (OPEN):** Installed First Voices Keyboard. Able to use the FV keyboard without having any problem.

File: android/KMEA/app/src/test/java/com/tavultesoft/kmea/util/FileUtilsTest.java
Comment (mcdurdin) [inline android/KMEA/app/src/test/java/com/tavultesoft/kmea/util/FileUtilsTest.java:32]: Why did this change with the refactor?

**Review by mcdurdin — APPROVED** [review]
LGTM. One question.

I note that you haven't renamed the test folders from tavultesoft.kmea but you have renamed the engine source folders.

File: android/KMEA/app/src/test/java/com/tavultesoft/kmea/util/FileUtilsTest.java
Comment (darcywong00) [reply android/KMEA/app/src/test/java/com/tavultesoft/kmea/util/FileUtilsTest.java:32]: I don't have a good answer for why that changed :/
The same lines did decrease when we recently bumped our dependencies in 2a0368bc54dcd333ad8758e4ef1a4748d952f97e

**Comment by darcywong00** [comment]
I'll wait on resolving the merge conflicts until all the localization PRs get merged

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.40-alpha](https://keyman.com/downloads/releases/alpha/17.0.40)

**Pull request merged**
